packages/core/src/types/address.ts

```diff
@@ -0,0 +1,6 @@
+export type Address =
```

Comment:
Missing some address types. Are we sure we want to be this strict?

Reply:
I would add the missing types.

Regarding strictness, what are the reasons to not be this strict? Viem was strict for evm addresses using `0x${string}

Reply:
I don't think anyone uses these types in their apps, so they will always end up casting it to our type. It's just easier to start with `string` for now and we can always make it stricter later.